metadata-ingestion/src/datahub/ingestion/source/gc/datahub_gc.py

```diff
@@ -281,6 +292,8 @@ def revoke_expired_tokens(self) -> None:
             list_access_tokens = expired_tokens_res.get("listAccessTokens", {})
             tokens = list_access_tokens.get("tokens", [])
             total = list_access_tokens.get("total", 0)
+            if tokens == []:
```

Comment:
Won't total be set to 0 if no tokens are returned ?

Reply:
It should be. But seems like we might have a bug in the backend.